src/RestApi/ProductReviews.php

```diff
@@ -0,0 +1,82 @@
+<?php
+/**
+ * REST API Reviews controller customized for Blocks.
+ *
+ * Handles requests to the /products/reviews endpoint. These endpoints allow read-only access to editors.
+ *
+ * @internal This API is used internally by the block post editor--it is still in flux. It should not be used outside of wc-blocks.
```

Comment:
`This API is used internally by the block post editor`

Should we update that to clarify it's also used in the frontend?